e2e/variantsTest.spec.js

```diff
@@ -0,0 +1,168 @@
+import { expect, test } from '@playwright/test'
+;[
+  {
+    pageAlt: 'morse',
+    buttons: [
+      [
+        '●',
+        '‒',
+        '●',
+        '/',
+        '‒',
+        '●',
+        '‒',
+        '/',
+        '‒',
+        '‒',
+        '/',
+        '●',
+        '●',
+        '‒',
+        '‒',
+        '/',
+        '‒',
+        '‒',
+        '‒',
+        '●',
+        '/',
+        '‒',
+        '●',
+        '●',
+        '●',
+        '/',
+        '‒',
+        '‒',
+        '‒',
+        '●',
+      ],
+    ],
+    variantId: 'Alternativní řešení ‒●/  ⇒  ●‒/',
+  },
+  {
+    pageAlt: 'braille',
+    buttons: [
+      ['2', '4', '5', '6'],
+      ['4', '6'],
+      ['1', '3', '5', '6'],
+      ['2', '4'],
+      ['4', '5'],
+      ['1', '3', '6'],
+      ['4', '5'],
+    ],
+    variantId: '123456-true',
+  },
+  {
+    pageAlt: 'semaphore',
+    buttons: [
+      ['1', '4'],
+      ['2', '6'],
+      ['1', '3'],
+      ['6', '7'],
+      ['4', '7'],
+      ['4', '6'],
+      ['4', '7'],
+    ],
+    variantId: '23456781',
+  },
+  {
+    pageAlt: 'ternary',
+    buttons: [
+      [
+        '0',
+        '2',
+        '1',
+        '1',
+        '2',
+        '2',
+        '0',
+        '2',
+        '2',
+        '1',
+        '1',
+        '1',
+        '1',
+        '0',
+        '0',
+        '0',
+        '2',
+        '0',
+        '1',
+        '0',
+        '0',
+      ],
+    ],
+    variantId:
+      'Trojkovka (hodnoty 0=1, 1=2, 2=0; pořadí 1.=>1., 2.=>2., 3.=>3.; A=0, s Ch)',
+  },
+  {
+    pageAlt: 'binary',
+    buttons: [
+      '1',
+      '0',
+      '1',
+      '0',
+      '0',
+      '0',
+      '1',
+      '1',
+      '0',
+      '1',
+      '1',
+      '0',
+      '1',
+      '1',
+      '0',
+      '0',
+      '0',
+      '1',
+      '0',
+      '1',
+      '0',
+      '1',
+      '0',
+      '0',
+      '1',
+      '1',
+      '0',
+      '1',
+      '0',
+      '1',
+      '0',
+      '1',
+      '0',
+      '0',
+      '1',
+    ],
+    variantId: 'Binár (hodnoty 0=1, 1=0; pořadí =>12345; A=1)',
+  },
+].forEach(({ pageAlt, buttons, variantId }) => {
+  test(`${pageAlt}, KRIZJ, as variant by buttons`, async ({ page }) => {
```

Comment:
Test case is named KRIZJ, but solution is KRIZS/KRIZVJV -- looks like inconsistency.